hv-rhel5.x/hv/tools/hv_vss_daemon.c

```diff
@@ -99,8 +100,10 @@ static int vss_operate(int operation)
 		return -1;
 
 	while ((ent = getmntent(mounts))) {
-		if (strncmp(ent->mnt_fsname, match, strlen(match)))
-			continue;
+		if (strncmp(ent->mnt_fsname, match_dev, strlen(match_dev)))
```

Comment:
There is actually a larger bug here that we need to fix. We should be skipping all "/dev/" devices.

The current code is not doing that since it's not checking for return of 0 in strncmp. Once we fix this, we shouldn't need to a separate "match_loop" comparison.